node/axiosConfig.ts

```diff
@@ -8,9 +8,8 @@ axios.interceptors.response.use(
       throw error
     }
 
-    const {config: {method}, status, url, data} = error.response
-    const responseData = typeof data === 'object' ? JSON.stringify(data) : data
-    const message = `External HTTP request failed. method=${method} status=${status} url=${url} data=${responseData}`
+    const {config: { method, url }, status, data} = error.response
+    const message = JSON.stringify({ method, status, url, data }, null, 2)
```

Comment:
I'd add some info so the console isn't only the JSON (which the user needs to guess that is there because it errored):
`External HTTP request failed with error: ${JSON.stringify(...)}`

Reply:
Fair enough